channel.go

```diff
@@ -888,3 +906,21 @@ func toStringSet(ss []string) map[string]struct{} {
 	}
 	return set
 }
+
+// take a list of service::method formatted string and make
+// the map[service]map[method]struct{} set
+func toServiceMethodSet(sms []string) (map[string]map[string]struct{}, error) {
+	set := map[string]map[string]struct{}{}
+	for _, sm := range sms {
+		s := strings.Split(sm, "::")
+		if len(s) != 2 {
+			return nil, fmt.Errorf(`Each "SkipHandlerMethods" value should be of service::Method format but got %q`, sm)
```

Comment:
nit: error should be lowercase (since they're often concatenated), and use `%q` for quoting.